frontend/src/app.js

```diff
@@ -21,8 +21,13 @@ const App = Vue.extend({
       }
     })
 
-    const darkMode = this.$localStorage.getItem('global/dark-mode') === 'true'
+    const darkMode = this.$localStorage.getItem('global/setting-dark-mode') || 2
```

Comment:
Why `global/seeting-dark-mode` and not `global/dark-mode` or `global/settings/dark-mode`?

Reply:
I would propose to have a localStorage porperty 'global/color-scheme' or  'global/settings/color-scheme' with possible values `dark`, `light` and `auto` or `no-preference`